1️⃣ 일정 추가 모달

seokjin925 opened the pull request:
모달 UI 구현 (제목, 동아리, 일시, 장소, 반복 옵션 포함)

반복 옵션 Select 드롭다운 적용

"추가" 버튼 클릭 시 일정 데이터 저장 기능
2️⃣ 통장 추가 모달

모달 UI 구현 (동아리, 계좌번호, 개설일, 소유주 포함)

개설일 DatePicker 적용

"추가" 버튼 클릭 시 계좌 데이터 저장 기능

seokjin925 committed:
1️⃣ 일정 추가 모달

모달 UI 구현 (제목, 동아리, 일시, 장소, 반복 옵션 포함)

반복 옵션 Select 드롭다운 적용

"추가" 버튼 클릭 시 일정 데이터 저장 기능
2️⃣ 통장 추가 모달

모달 UI 구현 (동아리, 계좌번호, 개설일, 소유주 포함)

개설일 DatePicker 적용

"추가" 버튼 클릭 시 계좌 데이터 저장 기능

mjgwon24 commented on src/components/common/modals/AddBankbook.jsx:
```diff
@@ -0,0 +1,81 @@
+    const handleSubmit = () => {
+        onSave(form);
+        onClose();
+    };
```
제출 전에 비어있는 필수 입력폼이 없도록 유효성 검사 기능도 이후 업데이트 사항에 추가해두면 좋을것같네용

mjgwon24 reviewed (commented):
확인했습니다!
직전 회의에서 다뤘던 Ant design 적용되면 확인 후 승인하도록 하겠습니다. 입력폼과 버튼이 ant 모듈로 변경되면 좋겠네요.

감사합니다!

mjgwon24 commented:
아 그리고 package json에 아래 의존성 추가 필요해보이네요. 업데이트 하실때 참고해주세요!

`
    "@emotion/react": "^11.14.0",
    "@emotion/styled": "^11.14.0",
`

seokjin925 committed:
fix: Select 드롭다운 높이 조정, 입력폼 검증, UI 개선

- 일정 추가 및 통장 추가 모달의 Select 드롭다운이 짤리는 문제 해결
- 드롭다운 최대 높이를 300px로 설정하여 스크롤 허용
- UI 일관성을 위해 모달 높이 조정 (800px, 650px)
- 기타 입력 필드 스타일 유지 및 오류 표시 개선
